content/en/docs/concepts/containers/container-environment.md

```diff
@@ -39,9 +39,11 @@ as are any environment variables specified statically in the Docker image.
 
 ### Cluster information
 
-A list of all services that were running when a Container was created is available to that Container as environment variables.
-This list is limited to services within the same namespace as the new Container's Pod and Kubernetes control plane services.
-Those environment variables match the syntax of Docker links.
+When the [Pod spec](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#pod-v1-core) field `enableServiceLinks` is `true` (the default),
```

Comment:
Hi. I would actually link to https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#containers - it's a newer API reference.